src/components/NSLookupTool/NSLookupTool.tsx

```diff
@@ -7,50 +7,54 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "NsLookup";
-const description_userguide =
-    "The nslookup command is a tool used to query Domain Name System (DNS) servers and retrieve information about a specific domain or IP address." +
+/**
+ * Title and Description of the tool
+ */
+const title = "NSLookup";
+const description_Userguide =
+    "The NSLookup command is a tool used to query Domain Name System (DNS) servers and retrieve information about a specific domain or IP address." +
     "This command is an essential tool for network administrators and system engineers as it can be used to troubleshoot DNS issues and gather information about DNS configurations." +
     "How to use NSLookUp.\n\n" +
     "Step 1: Enter an IP or Web URL.\n" +
-    "       E.g. 127.0.0.1\n\n" +
+    " E.g. 127.0.0.1\n\n" +
     "Step 2: View the Output block below to view the results of the Scan.";
 
-interface FormValues {
-    ipaddress: string;
+// Form Value Interface
+interface FormValuesType {
+    ipAddress: string; // Ip Address that needs to be looked up
+    tutorial: string; //Tutorial Text
 }
 
 export function NSLookup() {
-    const [loading, setLoading] = useState(false);
-    const [pid, setPid] = useState("");
-    const [output, setOutput] = useState("");
-    const [allowSave, setAllowSave] = useState(false);
-    const [hasSaved, setHasSaved] = useState(false);
+    // State the Variables
 
-    let form = useForm({
+    const [loading, setLoading] = useState(false); // Indication of running the process
+    const [pid, setPid] = useState(""); // Process ID
+    const [output, setOutput] = useState(""); // Output
+    const [allowSave, setAllowSave] = useState(false); // Looking whether the process can be saved
+    const [hasSaved, setHasSaved] = useState(false); // Indication of saved process
+
+    // Form management by using Mantine
```

Comment:
This is probably a more descriptive comment for this particular piece of code. 
```suggestion
  // Form Hook to handle form input.
```